shopping-app/src/components/cart/cartitem.js

```diff
@@ -0,0 +1,40 @@
+import { React } from 'react';
+import { listItems } from '../data';
```

Comment:
listItems should in a context and consumer `useContext` would be used here to get the data